jdaviz/core/template_mixin.py

```diff
@@ -270,7 +270,7 @@ def loaders(self):
         if not len(self.loader_items):
             self._update_loader_items()
         from ipywidgets.widgets import widget_serialization
-        if not (self.app.state.dev_loaders or self.config in ('deconfigged', 'specviz', 'specviz2d')):  # noqa
+        if not (self.app.state.dev_loaders or self.config in ('deconfigged', 'specviz', 'specviz2d', 'imviz')):  # noqa
```

Comment:
again, let's revert this if all tests don't pass and we decide to merge this under the dev-flag